src/lib.rs

```diff
@@ -171,7 +171,18 @@ impl<T> core::panic::RefUnwindSafe for Event<T> {}
 
 impl<T> fmt::Debug for Event<T> {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        f.write_str("Event { .. }")
+        match self.try_inner() {
+            Some(inner) => {
+                let notified_count = inner.notified.load(Ordering::Relaxed);
+                let total_count = inner.list.total_listeners();
+
+                f.debug_struct("Event")
+                    .field("listeners_notified", &notified_count)
+                    .field("listeners_total", &total_count)
+                    .finish()
+            }
+            None => f.debug_tuple("event").field(&"<uninitialized>").finish(),
```

Comment:
```suggestion
            None => f.debug_tuple("event").field(&format_args!("<uninitialized>")).finish(),
```

This makes it so the `<uninitialized>` text isn't in quotes.